src/Module/Contact/Profile.php

```diff
@@ -208,7 +208,7 @@ protected function content(array $request = []): string
 		$groups_widget = '';
 
 		if (!in_array($localRelationship->rel, [Contact::NOTHING, Contact::SELF])) {
-			$groups_widget = Group::sidebarWidget('contact', 'group', 'full', 'everyone', $contact['id']);
+			$groups_widget = Group::sidebarWidget('contact', 'group', 'full', 'everyone', $data['user']);
```

Comment:
Wouldn't it be better to check for the existance of `$data['user']`?

Reply:
The condition on line 210 implicitly ensures there's one.